specification/databox/DataboxApis.ts

```diff
@@ -112,6 +112,73 @@ export const DataboxApiSchemas = {
       error: z.any().optional(),
     }),
   },
+  'Databox.createInMemoryTable': {
```

Comment:
I haven't looked the Databox side, but do we really want these apis to all be these little units like this? What happens if one fails in a long set of changes? Is it an option to run all these internal commands in one "SetupSchema" type command?